.golangci.yml

```diff
@@ -103,7 +103,7 @@ linters:
     - tagliatelle # Checks the struct tags. [fast: true, auto-fix: false]
     - tenv # tenv is analyzer that detects using os.Setenv instead of t.Setenv since Go1.17 [fast: false, auto-fix: false]
     - testableexamples # linter checks if examples are testable (have an expected output) [fast: true, auto-fix: false]
-    # - testpackage # linter that makes you use a separate _test package [fast: true, auto-fix: false]
+    - testpackage # linter that makes you use a separate _test package [fast: true, auto-fix: false]
```

Comment:
Hay, do we really want this? I'm not 100% on this.

Reply:
if possible yes, cause it helps test your public api and you don't interfere with internal variables